packages/components/src/Lettera/Header.purs

```diff
@@ -0,0 +1,30 @@
+module Lettera.Header where
+
+import Prelude
+
+import Affjax.ResponseHeader as AX
+import Data.Array (catMaybes, find, fromFoldable)
+import Data.Int (fromString)
+import Data.Maybe (Maybe(..))
+import Data.String (Pattern(..), split, trim)
+
+type Directive =
+  { name :: String
+  , value :: Maybe Int
+  }
+
+parseCacheControl :: String -> Array Directive
```

Comment:
What does the cace control strings look like?

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control
I'm only using cache-control parsing for Lettera responses which we control so I'm not being overly aggressive with the parsing.